.github/workflows/e2e.yaml

```diff
@@ -418,14 +420,19 @@ jobs:
         env:
           TOGETHER_ENVIRONMENT: ${{ inputs.backendEnvironment }}
 
-      - name: Change node version to 12.18.0
+      - name: Change to frontend node version
         uses: actions/setup-node@v2
         with:
-          node-version: 12.18.0
+          node-version-file: './${{ inputs.frontendRepo }}/.nvmrc'
 
       - name: Start frontend
         run: yarn serve -s build -l 3000 & npx wait-on http://localhost:3000
         working-directory: ./${{ inputs.frontendRepo }}
+        
+      - name: Change node version to 12.18.0
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.18.0
```

Comment:
Have you tried updating the cypress action below to v3? I think they added support for Node 16: https://github.com/cypress-io/github-action/releases/tag/v3.0.0

Reply:
Certainly there is much more room for improvement here in this action. Lots of usage of node 12 still but I just want to unblock the update of the frontend for now. Perhaps the next step can be updating node in this action.

Reply:
No problemo. Just thought I'd call it out

Reply:
Appreciate it!